app/views/layouts/application.html.erb

```diff
@@ -8,7 +8,7 @@
     <meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport"/>
     <%= yield(:append_to_head) %>
     <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload", defer: true, nonce: true %>
+    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload", defer: !Rails.env.test?, nonce: true %>
```

Comment:
Because integration test with Cuprite can be very fast. First click can happen before javascript has started since by default JS parsing is `deferred`.

Here it fixes a flaky test for the collapse button.